truststore_linux.go

```diff
@@ -44,6 +44,10 @@ func init() {
 	} else if pathExists("/usr/share/pki/trust/anchors") {
 		SystemTrustFilename = "/usr/share/pki/trust/anchors/%s.pem"
 		SystemTrustCommand = []string{"update-ca-certificates"}
+	} else if pathExists("/etc/slackware-release") {
+		SystemTrustFilename = "/usr/share/ca-certificates/%s.crt"
```

Comment:
Does Slackware not support `/usr/local/share/ca-certificates`?

Reply:
Directory `/usr/local/share/ca-certificates` not exists by default and `mkcert` currently not create full path for `LOCALCERTSDIR`, use `/usr/share/ca-certificates` is a compromise.

Reply:
If `/usr/local/share/ca-certificates` is created, is it supported? If so I'm happy to change the code to create it, instead of mixing the mkcert root with the platform roots.

Reply:
Yes, `/usr/local/share/ca-certificates/` supported.